sxt/field12/operation/add.h

```diff
@@ -14,41 +14,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Empty components don't make sense

Reply:
I want to have a test for multi-limbed addition. To avoid cycles and get rid of a component that only has tests, I created an `element6` in the `base/field/example_element` component. The tests mirror the old `field12/add` tests.

If you don't like this, I think another solution would be to create a new package group just for Montgomery field operations. It can then avoid cycles between the base and field package groups while still testing multi-limbed addition. A Montgomery field package group might also keep the `base/field` package cleaner? Do you have a preference?

Reply:
Thinking about it more, It seems cleaner to have a new package for the Montgomery field operations. 

After the `field12/operation/neg` and `field12/operation/sub` components are made generic over the next two PRs, the `field_mtg` package group can then depend on `field12` without introducing a cycle. I can then implement multi-limbed addition tests without creating the new `element6` example element mentioned above.